### DISCUSSION TITLE
Solar Learning

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#673] [Q&A]
**Channel Name:** Solar Learning
**Channel ID:** SolarLearning.ph
**Country:** Philippines
**Language:** Tagalog/English
**External Link (optional):** xxx
**Notes (optional):** Free to air learning channel in the Philippines.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19671
